app/controllers/admin/conditions_controller.rb

```diff
@@ -0,0 +1,56 @@
+module Admin
+  class ConditionsController < BaseController
+    before_action :set_condition, only: %i[edit update destroy]
+    def new
+      @condition = Condition.new
+    end
+
+    def create
+      @condition = Condition.new(condition_params)
+      if @condition.save
+        flash[:success] = 'Condition successfully created.'
+        redirect_to @condition
+      else
+        flash.now[:danger] = 'Something went wrong. Please try again.'
+        render :new
+      end
+    end
+
+    def edit
```

Comment:
@iandouglas @AliSchlereth We were wondering whether we should omit empty methods like this or whether it's better to have them for ~rails magic~  purposes.

Reply:
@pollockcl keep this for readability reasons.